CMakeLists.txt

```diff
@@ -33,22 +33,23 @@ install(EXPORT ArborXTargets
   DESTINATION lib/cmake/ArborX
 )
 
-SET(ARBORX_VERSION_STRING "0.9 (dev)")
+SET(ARBORX_VERSION_STRING "0.9-dev)")
 SET(ARBORX_GIT_COMMIT_HASH "No hash available")
+
+
 IF(EXISTS ${PROJECT_SOURCE_DIR}/.git)
-    FIND_PACKAGE(Git)
-    IF(GIT_FOUND)
-        EXECUTE_PROCESS(
-            COMMAND           ${GIT_EXECUTABLE} log --pretty=format:%h -n 1
-            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-            OUTPUT_VARIABLE   ARBORX_GIT_COMMIT_HASH
-            )
-    ENDIF()
+  FIND_PACKAGE(Git)
+  IF(GIT_FOUND)
+    ADD_CUSTOM_TARGET(record_hash ALL
+                      COMMAND ${CMAKE_COMMAND}
+                        -DSOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}
+                        -DBINARY_DIR=${CMAKE_CURRENT_BINARY_DIR}
+                        -DARBORX_ENABLE_MPI=${ARBORX_ENABLE_MPI}
```

Comment:
What's the point of printing if it was compiled with `MPI`? I would rather get the version of `Kokkos` that was used.